ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml

```diff
@@ -277,122 +89,12 @@ tests:
       - mount_path: /tmp/secret/stackrox-scanner-secrets
         name: stackrox-scanner-secrets
         namespace: test-credentials
-      dependencies:
-      - env: SCANNER_SLIM_IMAGE
-        name: scanner-slim
-      - env: SCANNER_DB_SLIM_IMAGE
-        name: scanner-db-slim
       from: src
       resources:
         requests:
           cpu: 2000m
           memory: 4000Mi
   timeout: 3h0m0s
-- as: merge-store-db-dump
-  postsubmit: true
-  steps:
-    test:
-    - as: store-db-dump
-      commands: .openshift-ci/dispatch.sh store-db-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: db-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: merge-store-genesis-dump
-  postsubmit: true
-  steps:
-    test:
-    - as: store-genesis-dump
-      commands: .openshift-ci/dispatch.sh store-genesis-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: genesis-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: merge-style-checks
-  postsubmit: true
-  steps:
-    test:
-    - ref: stackrox-scanner
-- as: merge-unit-tests
-  postsubmit: true
-  steps:
-    test:
-    - ref: stackrox-scanner
-- as: merge-upload-db-dump
-  postsubmit: true
-  steps:
-    test:
-    - as: upload-db-dump
-      commands: .openshift-ci/dispatch.sh upload-db-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: db-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: merge-upload-dumps-for-downstream
```

Comment:
was this migrated?

Reply:
It should've been moved in this commit: https://github.com/stackrox/scanner/pull/1347/commits/97d5f4e49ba3d4eaa92a3630507200d09e08e7f7